res/css/photon/label.css

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+.photon-label {
+  display: inline-flex;
+  padding: 3px 0;
+  color: var(--grey-90);
```

Comment:
I see we have a `color: var(--ink-70)` in `.stackSettings`, I believe we can remove it. The label field for the search was inheriting it, but I think it's better if it gets the same `--grey-90` value for consistency, and good news! it actually inherits it from  `:root` once we remove the rule of `.stackSettings`.